apps/client/src/components/AddressDataStyles.js

```diff
@@ -1,7 +1,7 @@
 import styled from "styled-components";
 import { themeColor, themeSpacing } from "@datapunt/asc-ui";
 
-export const StyledAddressResult = styled.div`
+export const StyledAddressData = styled.div`
```

Comment:
Maybe AddressResult would be even better. but lets keep data :-)

Reply:
done